llvm-spirv/lib/SPIRV/SPIRVToOCL.cpp

```diff
@@ -537,8 +537,7 @@ llvm::createSPIRVBIsLoweringPass(Module &M,
   case SPIRV::BIsRepresentation::SPIRVFriendlyIR:
     // nothing to do, already done
     return nullptr;
-  default:
-    llvm_unreachable("Unsupported built-ins representation");
-    return nullptr;
```

Comment:
This change should be upstreamed to KhronosGroup/SPIRV-LLVM-Translator GitHub repo

Reply:
Do I need to revert this or leave it and upstream to Khronos repo too?

Reply:
Reverted those files. Thanks.